lib/Widgets/Utils.vala

```diff
@@ -290,6 +290,23 @@ public static Gdk.RGBA contrasting_foreground_color (Gdk.RGBA bg_color) {
     return fg_color;
 }
 
+/**
+ * Sets the app's icons, cursors, and stylesheet to elementary defaults.
+ */
+public void force_elementary_style () {
+    const string STYLESHEET_PREFIX = "io.elementary.stylesheet";
+
+    var gtk_settings = Gtk.Settings.get_default ();
+    var current_stylesheet = gtk_settings.gtk_theme_name;
```

Comment:
Do we need to cache this since we only use it once?

Reply:
@danrabbit ah yeah I think I used this when I was planning to do more but now it's a one-off. :+1: